apparmor.d/profiles-a-f/flameshot

```diff
@@ -35,13 +35,19 @@ profile flameshot @{exec_path} {
 
   /var/lib/dbus/machine-id r,
   /etc/machine-id r,
+  /usr/share/flameshot/translations/* r,
 
+  owner @{user_config_dirs}/dconf/user r,
   owner @{user_config_dirs}/flameshot/ rw,
   owner @{user_config_dirs}/flameshot/** rwlk -> @{user_config_dirs}/flameshot/**,
+  owner @{user_config_dirs}/ibus/bus/@{hex32}-unix-wayland-0 r,
```

Comment:
This should be `owner @{user_config_dirs}/ibus/bus/@{hex32}-unix-{,wayland-}@{int} r,`